datafusion/sqllogictest/src/engines/datafusion_engine/normalize.rs

```diff
@@ -198,12 +198,16 @@ macro_rules! get_row_value {
 ///
 /// Floating numbers are rounded to have a consistent representation with the Postgres runner.
 ///
-pub fn cell_to_string(col: &ArrayRef, row: usize) -> Result<String> {
+pub fn cell_to_string(
+    col: &ArrayRef,
+    row: usize,
+    data_type: &DataType,
```

Comment:
Minor nit is that since this data type comes from `col` it seems like we could keep the signature of this function the same and call `let data_type = col.data_type()` in the function

Reply:
oops... my bad. It's my other experimental change. I forgot to roll back it. I'll recover it. Thanks for mentioning it.